src/services/__tests__/messagesService.test.ts

```diff
@@ -263,6 +281,23 @@ describe("MessageService#getServicesByRecpient", () => {
     });
   });
 
+  it("returns an error if the getServicesByRecpient API returns an 429 HTTP error from the upstream API", async () => {
+    mockGetServicesByRecipient.mockImplementation(() =>
+      t.success(tooManyReqApiMessagesResponse)
+    );
+
+    const service = new MessageService(api);
+
+    expect.assertions(2);
+    return service.getServicesByRecipient(mockedUser).then(e => {
+      expect(mockGetServicesByRecipient).toHaveBeenCalledWith({
```

Comment:
no need to test this, we're testing just the return response

Reply:
Fix in e0e50dd